api/src/main/java/org/pmiops/workbench/exceptions/ExceptionUtils.java

```diff
@@ -31,6 +35,18 @@ public static RuntimeException convertGoogleIOException(IOException e) {
     throw new ServerErrorException(e);
   }
 
+  public static RuntimeException convertFirecloudException(ApiException e) {
+    if (e.getCode() == HttpServletResponse.SC_NOT_FOUND) {
+      throw new NotFoundException();
+    } else if (e.getCode() == HttpServletResponse.SC_FORBIDDEN) {
+      throw new ForbiddenException();
+    } else if (e.getCode() == HttpServletResponse.SC_SERVICE_UNAVAILABLE) {
+      throw new ServerUnavailableException();
+    } else {
+      throw new ServerErrorException();
```

Comment:
The previous code included `e.getResponseBody()`. Not sure I actually agree with doing that, but noting the difference to check whether this was intentional

Reply:
Yeah, I don't think we need to include that here.